### ISSUE TITLE
Feature request: Use `@media print`

### ISSUE THREAD

**Issue by selfagency** [eKoopmans/html2pdf.js#339]
Great tool and I thank you for it. One minor request, if you ever find the time and have an idea for how to accomplish it:

It would be great if the renderer respected the `@media print` stylesheet settings so that you can control which elements appear in the PDF via CSS.

**Comment by gillespieza** [comment]
+1 for this feature request

**Comment by roddk939** [comment]
Agreed. This is 1 of only 2 issues I have. Great simple javascript plugin.

**Comment by tariqulislam16** [comment]
Such feature will take this library to another level. Waiting for this feature to be added.

**Comment by Daftnap** [comment]
Also waiting for this feature. Invoices are printed dark-based on my PDF when using my app dark mode. It would be great to "isolated" css rules related to the use of html2pdf.js

**Comment by vengi83644** [comment]
+1
